easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2112-foss-2021b.eb

```diff
@@ -8,7 +8,10 @@ description = """OpenFOAM is a free, open source CFD software package.
  to solid dynamics and electromagnetics."""
 
 toolchain = {'name': 'foss', 'version': '2021b'}
-toolchainopts = {'cstd': 'c++11'}
+# Users have found that vectorizion caused OpenFOAM to produce some very incorrect results.
+# Disabling vectorize was confirmed to fix the the known issues.
+# With no test suite, sticking to known working toolchain options until proven otherwise.
```

Comment:
is there an issue open for this upstream?

Reply:
No, at least not yet, i think.
This is the results of length email discussion with a professor in fluid dynamics and long time OpenFOAM user at my university, Håkan Nilsson (not sure he has a github account).
I think he is planning to open a issue with them.

I build several variants and he wrote a summary of everything tested:
```
OpenFOAM/v2112-novectorize-precise:
PASS
OpenFOAM/v2112-precise:
FAIL
OpenFOAM/v2112 (present installation, EasyBuild with -novectorize):
PASS
OpenFOAM/v2112 (first installation, default EasyBuild settings):
FAIL
My own standard OpenFOAM installation with -O3 (using foss-2021b):
PASS
My own standard OpenFOAM installation with -O3 -march=native (using foss-2021b):
PASS
My own standard OpenFOAM installation with default EasyBuild flags (using foss-2021b):
FAIL
My own standard OpenFOAM installation with default EasyBuild flags (using foss-2020a, since that was used for working OpenFOAM/v2006):
FAIL


Other system-installed (EasyBuild) versions:
OpenFOAM/v2006
PASS
OpenFOAM/v1912
PASS
```

This was on Skylake server architecture.

Reply:
@Micket since `-O3` seems to give valid results, perhaps we should just move to that instead of disabling vectorization?